lib/perl/Genome/Model/ReferenceAlignment/Command/RefCovMetrics.pm

```diff
@@ -0,0 +1,124 @@
+package Genome::Model::ReferenceAlignment::Command::RefCovMetrics;
+
+use strict;
+use warnings 'FATAL';
+
+use Genome;
+
+require List::MoreUtils;
+
+class Genome::Model::ReferenceAlignment::Command::RefCovMetrics {
+    is => 'Command::V2',
+    has_input => {
+        models => {
+            is => 'Genome::Model::ReferenceAlignment',
```

Comment:
That's even more generic than I was thinking of making this, but that'd be fine by me.  The object that was used to do the query could be listed in the row, but the result's ID would have to be the primary identifier.

It's enough to me if the command works on both reference alignments and somatic validation, since those are the primary two model types used to look at coverage.  (Or, alternatively, there's a second command in the `genome model somatic-validation` namespace that does the same thing as this for those models.)